doc/cephadm/certmgr.rst

```diff
@@ -103,54 +111,54 @@ Cephadm certmgr supports three different scopes for certificate management:
 
 1. **Global Scope:**
    - Certificates in this scope are shared across all service daemons, regardless of which host they are running on.
-   - Example: `mgmt-gateway` certificate is a globally shared certificate used by all service daemons.
+   - Example: ``mgmt-gateway`` certificate is a globally shared certificate used by all service daemons.
 
 2. **Per-Host Scope:**
    - Certificates are assigned per host, meaning each host has its own unique certificate.
    - When configuring a custom certificate, the user must specify the host for which the certificate applies.
-   - Example: `grafana` service cerificates are configured at the host level and applies specifically to a single machine.
+   - Example: ``grafana`` service certificates are configured at the host level and apply specifically to a single machine.
 
 3. **Per-Service Scope:**
-   - Certificates are configured per service name, meaning each instance of a service can have its own certificate.
+   - Certificates are configured per service name (each service instance can have its own certificate).
    - When specifying a custom certificate, the user must define the service to which it belongs.
-   - Example: an `rgw` service certificate is assigned specifically and only to an RGW service
+   - Example: an ``rgw`` service certificate is assigned specifically and only to an RGW service
```

Comment:
An empty line is needed before the unordered sub-lists (between lines 113-114, 116-117, 121-122), right now they are just rendered as single paragraphs and not as unordered lists.